### ISSUE TITLE
C++ codegen: SpinCtrlDouble generates .SetDigits instead of ->SetDigits

### ISSUE THREAD

**Issue by ProgrammierPatrick** [wxGlade/wxGlade#497]
The C++ codegenerator generates:
```c
spin_ctrl_double_1 = new wxSpinCtrlDouble(panel_6, wxID_ANY, wxT("0.0"), wxDefaultPosition, wxDefaultSize, wxSP_ARROW_KEYS, 0.0, 100.0);
spin_ctrl_double_1.SetDigits(2);
grid_sizer_1->Add(spin_ctrl_double_1, 0, 0, 0);
```
, which should be correctly:

```c
spin_ctrl_double_1 = new wxSpinCtrlDouble(panel_6, wxID_ANY, wxT("0.0"), wxDefaultPosition, wxDefaultSize, wxSP_ARROW_KEYS, 0.0, 100.0);
spin_ctrl_double_1->SetDigits(2);
grid_sizer_1->Add(spin_ctrl_double_1, 0, 0, 0);
```

I have fixed it locally by just replacing two format strings in widgets/spin_ctrl_double/codegen.py:

- `'%s->SetIncrement(%s);\n'` instead of `'%s.SetIncrement(%s);\n'`
- `'%s->SetDigits(%s);\n'` instead of `'%s.SetDigits(%s);\n'`

**Comment by DietmarSchwertberger** [comment]
Thanks a lot. The modifications and also test cases are in the repository now.

**Comment by DietmarSchwertberger** [comment]
Released v1.0.3
